### ISSUE TITLE
Add meta information for the indexer

### ISSUE THREAD

**Issue by marble** [TYPO3-Documentation/t3SphinxThemeRtd#58]
Information like this should be in the HTML.
Using META-tag in the HEAD-section would be fine.

```
<div id="book-information">
   <div id="book-name">blog</div>
   <div id="book-version">8.7.0</div>
   <div id="book-type">Extension Manual</div>
</div>
```

**Comment by wmdbMattes** [comment]
`<meta name="book-name" content="blog">`
`<meta name="book-version" content="8.7.0">`
`<meta name="book-type" content="Extension Manual">`

**Comment by marble** [comment]
Let's keep the Slack content here as well:

Martin Bless [11:34 PM] 
@mathias.schreiber wants to have meta tags in the docs for the indexer of the global search. Example:
```<meta name="book-name" content="blog">
<meta name="book-version" content="8.7.0">
<meta name="book-type" content="Extension Manual">
```
That brings up the question what names we want to agree on for the "book-type".
What about this names (first column is important):
```=================  =============================================
BOOK_TYPE          DESCRIPTION
=================  =============================================
Blogpost           Blogpost, opinion, I case we have that somewhen.
Book               Is a book or part of a book
Cheat Sheet        Summarized help notes about various topics.
Extension Manual   Manual of a TYPO3 CMS extension.
Guide              In-depth exploration of specific topics.
Prototype Manual   Structure and markup example for official manuals and extension manuals. 
Reference Manual   Exhaustive listings of properties, options, APIs, etc.
Snippet            For snippets, tips, howtos. A howto is much shorter than a tutorial.
Team Notes
Tutorial           Introductory manual into the world of TYPO3 CMS.
=================  =============================================
```
(edited)


[11:35] 
Cu tomorrow.

**Comment by marble** [comment]
Implemented with v3.6.14
